docs/BLTouch.md

```diff
@@ -1,50 +1,129 @@
+the probe is exactly located. Lets start with the `x_offset` and the `y_offset`
+
+In order to find the proper vertical offset of the probe you need to know a certain point of
 your bed. X/Y = 0 may be a good point for this, any other will do as long as you know it.
-Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removeable mark on your
+Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removable mark on your
 bed by i.e. using a non permanent marker.
 
-Now move the tip of the BL-Touch over that point by jogging there with the conrols in the `control` tab of
+Now move the tip of the BL-Touch over that point by jogging there with the controls in the `control` tab of
 OctoPrint. Once the BL-Touch is roughly over the point acquire it using the `GET_POSITION` command.
 The difference to your marked point is your `x_offset` and `y_offset` to configure in the `printer.cfg`.
 
 
+### Z Offset
+
+1. We start by changing the `z_offset` in the `bltouch` section of the configuration to 10. This
+setting is very wrong, but it will make it possible to move down to the right height using the
+menus. Run `RESTART` after changing it to reload the configuration.
+
+2. Now run `G28`, to home the printer. Take a note of the point where the pin hits the bed and use
+the menu to move the nozzle to that point. Now take a *folded* paper and put it under the nozzle.
+Move the nozzle down using the menu, in 0.1mm steps, until it grabs the paper. It doesn't matter how
+much it grabs, you are searching for the higest position that grabs the paper. The nozzle should now
+be between 0.1mm and 0.2mm above the printbed. If you wonder why we used a folded paper, the answer
+is that the granularity of the movement is only 0.1mm, or the same as a normal paper thickness, that
+means that you could easily move too far and hit the bed.
+
+3. Now update the `z_offset` to reflect this, by substracting the current offset (10) by the reading
+on the printer display. So if it says 8.2, then `10-8.2` = 1.8
+
+4. Restart the printer, issue a `G28`, move to the pin location, then `G1 Z0`, and verify that it's
+still grabbing the paper as it was before. If not, then repeat step 2. and 3. until you are
+satisfied.
+
+5. At this point it's a good idea to verify that the offset is close to 1mm, if not, then you
+probably want to move the probe up or down to fix this. You want it to trigger well before the
+nozzle hits the bed, so that possible stuck filament or a warped bed don't affect any probing
+action. But at the same time, you want that the retracted position is as far above the nozzle as
+possible, to avoid damage by touching printed parts.
+
+6. Now it's time to fine tune with a real print. First make sure that your flow rate and steps are
+calibrated properly, so that wrongly configured flow rates don't affect the calibration. Then slice
+a one layer thick object, located close to where you are homing. I recommend a simple 50x50 mm
```

Comment:
shouldn't this rather reference the calibration chapters? i.e. the delta chapter contains a probing figure

Reply:
I'm not very familiar with delta printers, but yes, you are probably right that we should reference to the delta calibration guide here for that. If I understand how they work correctly, the precise z-offset probably doesn't matter, since it's implicitly calibrated when performing the delta calibration.

For other printers, I don't think we have a suitable guide for z-offset calibration, so I leave my instructions.

Another thing that I should probably at least mention, is that there are different ways of doing this. Some people use calipers to measure the thickness, although I believe that it's not precise enough. A micrometer might be, but I'm not fully sure about even that, since the plastic can easily have some small blobs that messes the measurements up. I'm able to visually tell the difference with even less than 0.01mm using my method, so the measurements would have to be more accurate than that. Another way of doing it is to just calibrate how well the filament sticks to the bed, some bed surfaces and filaments might actually need an "incorrect" z-offset in order to make it stick or to make it not stick too hard.

Reply:
Ok, sounds reasonable. Probably one would rather take a round shape for that on the delta, since the bed is round. So maybe write `resembles the shape of your bed` ?

Reply:
I don't think that's necessary, if we instructs delta printer users to use the delta calibration guide, and skip the fine tuning step completely.

The shape does also not really matter, but a square has long straight lines, which makes it easier to see the changes. Setting the infill direction parallel to square (or rotate it 45 degrees) would make it even better, since then each line would be the exact same length.

Reply:
how about a one liner of openscad producing such a figure?

Reply:
I started to think more about the delta printers, and now I'm not sure what's the correct order of things are at all, and what you actually need to do. I guess I have to get familiar with how they work first, and have someone with an actual delta printer to review the sequence. Or maybe it would be even easier if someone with a delta printer, wrote the procedure for it.

For example, I don't think we can do even the x/y calibration process without first having a calibrated delta printer, which moves the correct distances. I'm also not sure how even the basic initial paper calibration would work.